tests/chrono_binding.cpp

```diff
@@ -0,0 +1,41 @@
+#include <sqlite_orm/sqlite_orm.h>
+#include <catch2/catch.hpp>
+
+using namespace sqlite_orm;
+
+#if defined(SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED)
+
+TEST_CASE("sys_days_as_bindable_column_type") {
+
+    struct Person {
+        int id;
+        std::string name;
+        std::chrono::sys_days birthdate;
+    };
+
+    const std::string db_name = "sys_days.sqlite";
+    ::remove(db_name.c_str());
+
+    auto storage = make_storage(db_name,
+                                make_table("Persons",
+                                           make_column("id", &Person::id, primary_key(), autoincrement()),
+                                           make_column("name", &Person::name),
+                                           make_column("birthdate", &Person::birthdate)));
+
+    SECTION("YMD_sys_date") {
```

Comment:
nit: no need to add section if it is only one. Sections are helpful if you have to split the scenario

Reply:
I leave it open for extension that's why I put a SECTION()...